wca/resctrl_allocations.py

```diff
@@ -385,13 +387,30 @@ def validate_l3_string(l3, platform_sockets, rdt_cbm_mask, rdt_min_cbm_bits):
                        rdt_min_cbm_bits)
 
 
-def validate_mb_string(mb, platform_sockets):
+def validate_mb_string(mb, platform_sockets, mb_min_bandwidth):
+    assert mb_min_bandwidth is not None
     if not mb.startswith('MB:'):
         raise InvalidAllocations(
             'mb resources setting should start with "MB:" prefix (got %r)' % mb)
+
     domains = _parse_schemata_file_row(mb)
     _validate_domains(domains, platform_sockets)
 
+    for mb_value in domains.values():
+        check_mb_value(mb_value, mb_min_bandwidth)
+
+
+def check_mb_value(mb_value: str, mb_min_bandwidth):
+    try:
+        mb_value = int(mb_value)
+    except ValueError:
+        raise InvalidAllocations("{} is not integer format".format(mb_value))
+
+    if mb_value < mb_min_bandwidth:
+        raise InvalidAllocations(
+                "{} mb value. Requires minimum {} bandwidth"
```

Comment:
```suggestion
                "mb allocation smaller than minimum value {}"
```
I think we dont need to print  the actual value being set